{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings.py

```diff
@@ -168,3 +169,11 @@
 
 # This is needed to get a CRSF token in /admin
 ANON_ALWAYS = True
+
+USE_X_FORWARDED_HOST = config('USE_X_FORWARDED_HOST', default=True, cast=bool)
```

Comment:
Not directly related to security middleware but it's security feature (for hosts verification). Set the default to `False` to match Django's default and added docs.